src/peft/utils/other.py

```diff
@@ -655,6 +675,18 @@ def unload_and_optionally_merge_module(
         return self.token_adapter.get_base_layer()
 
 
+def _get_input_embeddings_name(model):
```

Comment:
The function could get a `default` argument that it returns instead of `None`, like `getattr`, but no strong opinion.